src/lib/auth/update-user-reset-password-token.js

```diff
@@ -0,0 +1,38 @@
+import query from '../../db/query';
+import { updateResetToken as queryText } from '../../db/sql-queries.json';
+
+/**
+ * updateUserResetPasswordToken updates user with resetPasswordToken and resetPasswordExpires
```

Comment:
maybe the function could just be `updateResetPasswordToken` just to keep it from being too long?